firebase-components/firebase-dynamic-module-support/gradle.properties

```diff
@@ -12,5 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-version=16.0.1
+version=17.0.0
```

Comment:
This hasn't move from beta03 in ages, can be left as is

https://maven.google.com/web/index.html?q=dynamic#com.google.firebase:firebase-dynamic-module-support